discord/interactions.py

```diff
@@ -303,10 +303,27 @@ async def original_message(self) -> InteractionMessage:
         )
         state = _InteractionMessageState(self, self._state)
         message = InteractionMessage(state=state, channel=channel, data=data)  # type: ignore
-        self._original_message = message
+        self._original_response = message
         return message
 
-    async def edit_original_message(
+    async def original_message(self):
```

Comment:
Perhaps this and `edit_original_message` could be deprecated

Reply:
That is what I had done originally, until @EmreTech suggested that they should be added as aliases.

Reply:
> That is what I had done originally, until @EmreTech suggested that they should be added as aliases.

Yeah I should've maybe elaborated that the aliases should "raise" a FutureWarning indicating that they are deprecated
Sorry about that

Reply:
Because if we didn't have aliases this would be a big breaking change and would be more suited for the next major version (v3.0)